pyanaconda/display.py

```diff
@@ -121,13 +120,14 @@ def ask_vnc_question(anaconda, vnc_server, message):
     spoke = AskVNCSpoke(anaconda.ksdata, message=message)
     ScreenHandler.schedule_screen(spoke)
     App.run()
+    ui_proxy = RUNTIME.get_proxy(USER_INTERFACE)
 
-    if anaconda.ksdata.vnc.enabled:
+    if ui_proxy.Vnc.enabled:
```

Comment:
I would store the variable locally because I have a bad feeling that this way you are doing unnecessary calls on DBus for each read and these could even change in the middle of the execution of this code.